### DISCUSSION TITLE
[Discussion] Regarding the 4chan raid

### DISCUSSION THREAD

**Discussion by ghost** [tenacityteam/tenacity-legacy#111] [General]
Hi all, again, no harm intended. I figured it was more appropriate to re-post my issue from another thread. I improved some bits as well so it's not a true word for word repost. I want to be more open minded on this attempt, so please bare with me.

---

**Summary:** Audacity has spyware or whatever blah blah. Many people fork it, this one becomes popular. Anon makes a thread to vote for sneedacity, one of the options on the original poll, no harm is committed in the action. Cookiengineer doesn't like this, despite it being an option. He makes a new thread, deleting any mention of sneedacity. Hell breaks loose. He *obviously* starts making posts on the board, but I cannot backup this claim. He claims he is harassed personally, yet no evidence or proof of calls or such is noted at all. He modifies his bio to slurs and such. Now he steps down, maybe for the best of all this.

Alongside this, there has been a lot of FUD and misinformation going around, and it's giving the folks who legitimately voted on sneedacity a bad reputation 

First off, my friend made the original thread and we discussed it. We had NO malicious intent regarding this "raid" or "trolling", we simply wanted to poke fun at a funny option. Everyone would've been fine, had the raid been known earlier and acknowledged, cookiengineer would've chosen the second best option and moved on. Instead, this entire situation turned into a flame war and is giving many a bad reputation. Now there is FUD spreading around too on hackernews and various other sites. 

So People found it funny, and everyone voted as normal. No harm was intended. And **importantly, nobody was called or emailed in this situation**, especially since there is no proof whatsoever (and claims to back up this). To those that did derailed the original purpose of what my friend did, shame on you. Sneedacity WAS an option, does it even make sense? Why was it there to begin with? People chose it, but since the word "4chan" was involved and fox news gave many the controversial impression of the site, this turned into a bigger matter. 

The calls likely didn't happen nonetheless. A tweet was made that cookiengineer (I'll call them Cookie for now) received 600 calls or such and his family got contacted, yet there is no proof that any of this happened, and it sounds EXTREMELY illegitimate. First off, 600 calls is absolutely insane, there weren't doubtably that many people participated in the thread alone when the tweet was made. Nobody on /g/ ever mentioned phoning or any information regarding them. **UNTIL** there is proof disclaiming this debate for the now deleted tweet, I can't believe many people would act, since after lots of digging around, nobody could find his phone number. Yes, /g/ has annoying shitstains but that is due to the flame war nature of the event, this was never intended anyway. 

(**Added on**) So now, he's claiming to false flag the sneedacity repository. The sneedacity repository did not break any rules within Github's policy, and this action is against open source in general. Sneedacity was it's own road, a fork, and did not want to cause harm.

(**Added on**) And, irony drops the cat, his bio changes continuously to slurs, random posts from the threads (despite him denying he interacted with the threads, he did, there was an archive somewhere). He claims somebody looked into his house; he also has made various misinformed claims of getting 600 messages and such (now deleted), which biases out, but this is up to you to decide.

Then, people started claiming sneed meant "special needs", which is just absolutely delusional. Sneed was a simpsons joke, so if that's the case, feel free to cancel them. This assumption is from urban dictionary and it has more dislikes than likes, since it's wrong and never used. If this was the case, then I could say that "kids" means "kitten with aids." 

Less related, but someone assumed 4chan was involved in the KiwiFarms death of an snes dev. This is NOT true, and majority of the posts on the thread on /g/ (now archived, someone find a link) stated sentences like "Fuck KF" and similar. 4chan hates kiwifarms and wants nothing to do with it. Even 8chan(kun now) was fed up with the shit and is why kiwifarms was born. 

Finally, Cookie said he wanted to ban any trolls from contributing kind of "people with anime pics." yet sneedacity has contributed more actual code to the repository then this one from people with anime pfps. Doesn't this seem a bit wrong? 

Anyway, I'm feed up with this stuff. It was a small simple joke, and Cookie took it to an extreme and it went over his head. Now he's blaming mostly the wrong people; all of GH, twitter, and 4chan is wrong. This is nonsense, but it seems to be over now, I hope.

**Comment by ghost** [comment]
Sources (unnamed):
- https://desuarchive.org/g/thread/82402134/
- https://desuarchive.org/g/thread/82405601/ 
- https://desuarchive.org/g/thread/82407520/
- https://desuarchive.org/g/thread/82409026/
- https://desuarchive.org/g/thread/82410116/
- https://desuarchive.org/g/thread/82411441/
- https://desuarchive.org/g/thread/82413937/
- https://desuarchive.org/g/thread/82416433/
- https://desuarchive.org/g/thread/82418711/
- (he posted a lot on these ones)
- https://desuarchive.org/g/thread/82420783
-  https://desuarchive.org/g/thread/82423292/
- https://web.archive.org/web/20210706000825/https://twitter.com/cookiengineer/status/1412201646378131459
- https://archive.is/ZqK2T

**Comment by differentprogramming** [comment]
Bullshit.
Sneedcity is a 4chan troll fork, and that's 100% clear looking at it.

**Reply by differentprogramming** [reply]
BTW, is 4chan Muse's personal army?

**Reply by port19x** [reply]
We all hate Muse, thats why Forks are popping up in the first place

**Reply by ghost** [reply]
Weird, because it's currently got the most contributions and discussion so far relating to actual code. (I'm still trying to learn the codebase, though, so I'm testing things)

**Reply by differentprogramming** [reply]
Contributions like what?  Learning how to install build tools?

**Reply by ghost** [reply]
> Contributions like what?  Learning how to install build tools?

Partially yes, they've moving away from conan due to it being a pain to maintain and old. Not to mention this project is more focused on how to rename and moderating the issues ;)

**Reply by differentprogramming** [reply]
People are going to have a lot of trouble trusting a 4chan project.
What happened to cookieengineer was mean, but yeah, going back to the beginning of 4chan the joke about Anonymous is that in any anonymous crowd there's a few assholes.

But you know, the way you all talk now, calling everything evil "based" it sounds like everyone is TRYING to be the asshole.

**Reply by ghost** [reply]
Tox originated from 4chan, as did several other little projects. This isn't a first

**Comment by AndrePuls** [comment]
I just stumbled over all this about this and I have no idea who is to blame for what. But your post is a near-perfect admission of guilt, that is only beaten by Donald Jodokus and Eric "Big Brain" Trump.

> no harm intended
> no harm is committed in the action
> We had NO malicious intent regarding this "raid" or "trolling"
> No harm was intended
> this was never intended anyway.
> we simply wanted to poke fun at a funny option.
> So People found it funny

If someone has to repeat his "good intentions" THAT often, then his ACTIONS were pretty bad. Last week, I INTENDED to follow the curved road. But instead, I was drunk and killed 17 people. But hey, intentions were good. No harm intended. Police and the families of the victims didn't buy it.

> Everyone would've been fine, had the raid been known earlier

So it's our fault. Got it.

> The sneedacity repository did not break any rules within Github's policy, and this action is against open source in general

Lousy argument. I didn't break Github's policy either, when I wiped out half of the city, last week,. But I don't consider it funny, though. In fact, it was the minority(!!!) of children of 4chan, who didn't get their "funny" option and raided this repository.. When bullies begin to harass a legitimate fork of a software project to force their will onto others, THAT is against Free and Open Source, you Big Brain. Living in a Democracy does NOT mean YOU decide. It means, you have to follow the social contract for a friendly and humane society. It also means, that you have to accept the opinion of other people. Especially, if they are the ones ultimately in charge of the project. If you are unhappy with that, make a fork and try to do a better job. Again: "Raiding" a repository, because you didn't get your "fuuny" name is WRONG. 

>  and everyone voted as normal.. And **importantly, nobody was called or emailed in this situation**, especially since there is no proof whatsoever (and claims to back up this).

Why didn't everyone vote "normally" right from the beginning? Fun fact: I continued to drive "normally" after I extinguished half of a continent. So it's fine. And "funny". And by "fine" I mean "not".
I have no idea who you are, and you have no idea who I am and why I involve myself here. But I know one thing: I am nowhere near HIS phone. Are you? No? Then how can you claim that didn't happen? To use your words: There is no claim, that didn't happen. Pretty weak argument, if you ask me. Especially since it was you and your people raiding HIS repository.

> To those that did derailed the original purpose of what my friend did, shame on you.

So things DID get out of hand. Interesting. Again: "Intent" does not matter if actual harassment and harm was done. I could mention my driving skillz here, but it gets old and not funny by now. You nprobably don't get it anyway.
 
> Sneedacity WAS an option

And it was rejected. Deal with it like a grown-up.

> does it even make sense? Why was it there to begin with? People chose it,

As a joke. Because it is "funny". You said it yourself

> but since the word "4chan" was involved and fox news gave many the controversial impression of the site, this turned into a bigger matter.

My friend, that is the most ignorant and oblivious thing I have read for a very long time. I don't watch Fox News (different continent, thank God!). And even I know, 4chan is a group of mentally deficient children, who throw a temper tantrum, if they don't get their will. Your bad reputation is DIRECTLY CAUSED by the actions of the bad people there. Actions, that are not tolerated in any other forum. That is not the fault of Fox News. In fact, I don't want want to be connected with bad behaviour, too. But my solutionm is - simpole enough - NOT acting badly. Most of the time. Only when I drive my car and destroy a small solar system. Okay, medium sized. But ugly! So it's fine. And I was the only one who showed up in intergalactic court, since everyone else was dead. So it worked out for me. By the way, even Fox News said, that their "opinions" should not be taken seriously. Yes, they really did that. But the actions and the racist and horrible opinions of 4chan are well known and documented. And you complain about Cookies allegedly racist comments, that conveniently got deleted. Your ignorance is out of this world. Unbelievable.

> First off, 600 calls is absolutely insane, there weren't doubtably that many people participated in the thread alone when the tweet was made.

You make the assumption, that people are incapable of making more than one call ever. News flash: Technology exists. Computer can be programmed. Robocalls, speed-dialing and Automation are a thing for over 30 years.

> The calls likely didn't happen nonetheless. A tweet was made that cookiengineer (I'll call them Cookie for now) received 600 calls or such and his family got contacted, yet there is no proof that any of this happened, and it sounds EXTREMELY illegitimate.Nobody on /g/ ever mentioned phoning or any information regarding them. **UNTIL** there is proof disclaiming this debate for the now deleted tweet, I can't believe many people would act, since after lots of digging around, nobody could find his phone number. Yes, /g/ has annoying shitstains but that is due to the flame war nature of the event,

Couldn't have said it better myself. Shitstains. You suck at being a lawyer. And yet, you make the claim, they didn't do any shitty things." Good intentions", I know. By DEMANDING "proof", the way you do, you shift the  burden of proof  onto the victim, which is not necessarily wrong. But there is also a thing called "plausibility". And your "shitstain" covered side, that raided this repo, because they didn't get thgeir "funny" will, has a long, long history of being assholes. So I simply tend to believe Cookie. He has nothing to gain by making false claims, in fact, he gave up his leading position here. You guys just wanted to have "fun". It fits like a glove.
 
> (**Added on**) So now, he's claiming to false flag the sneedacity repository.  Sneedacity was it's own road, a fork, and did not want to cause harm.

Again, didn't "want".

> He claims somebody looked into his house; he also has made various misinformed claims of getting 600 messages

And I believe him. How about, iinstead of repeating "Naa, haa!", you explain, how YOU can prove a NEGATIVE and why we should take YOUR word over his? Are YOU near his house ALL THE TIME? And was it YOU, who called him 550+ times, "just to check, if the line is busy, because someone else is supposedly calling him"? Dude, In that case, YOU are the problem. And there is NO CLAIM OR PROOF, that it WAS NOT YOU. What a stoopid, stoopid way of argumenting you have.

> Then, people started claiming sneed meant "special needs", which is just absolutely delusional. Sneed was a simpsons joke, so if that's the case, feel free to cancel them. This assumption is from urban dictionary and it has more dislikes than likes, since it's wrong and never used.

Soo.... it is okay, because a well researched dictionary is wrong, because a cartoon show on TV made a joke about it...? You can't mean this as a serious argument. I'm not a native english speaker and I haven't heard that word before and don't the meaning (or forgot). But when even the dictionary mentions this word iin a negative context, then it IS a VERY BAD CHOICE for a Free, Open and Inclusive software project with millions of users and hundreds of contributers. Thanks for making that argument against your own case. Just because you are not aware of the negative context and like cartoons, doesn't make it the right choice. Not even close to a "funny" one. "Gay" meant "happy" back in the days. Language is a living and ever changing thing. And by the way, only guilty and bad people have the need to change the context of their actions the way you do. And ONCE AGAIN: Living in a democracy DOES NOT MEAN that YOU can bully your opinion onto others.

> If this was the case, then I could say that "kids" means "kitten with aids."

Wow. Another example of a sick mind. What else do you find "funny"? Something about Jews, by chance? You must be a troll. Or a asshole. Probably both. And a idiot.

> Less related, but someone assumed 4chan was involved in the KiwiFarms death of an snes dev. This is NOT true, and majority of the posts on the thread on /g/ (now archived, someone find a link) stated sentences like "Fuck KF" and similar. 4chan hates kiwifarms and wants nothing to do with it. Even 8chan(kun now) was fed up with the shit and is why kiwifarms was born.

No clue what that is about. But I learned, that the shitty behaviour of 4chan - which is driven by "good intentions" - caused a farm in New-Sealand, but hates it - again, in  "good intentions", I presume. And they were linked to a death. That does not seem like "wants nothing to do with it". That sounds like directly involved. You know what? I have newer been linked to the death of someone. Being linked to bad behaviour doesn't happen by chance. It is usually justified. Espacially, when it's happening over and over and over and over and over and over again.

> Finally, Cookie said he wanted to ban any trolls from contributing kind of "people with anime pics." yet sneedacity has contributed more actual code to the repository then this one from people with anime pfps. Doesn't this seem a bit wrong?

No, it doesn't. Banning trolls is actually the exact RIGHT thing to do. Trolls, who just want to cause chaos "for fun" get banned in most places on the internet all the time. Places with an intact sense of morilty, that it. And it doesn't matter, how much code those trolls contribute. You caused Cookie to step back from his position. A murderer is a murderer, even if he once accidentally gave 5 bucks to a homeless person. Or in words, that you probably understand better: That "funny", little man with a mustache, who was pretty angry and ruled Germany for a short time, once invented Teflon and the Autobahn. That still makes him, his ideas, behaviour, intentions, humor and moral one of the most depravited things, that have ever been on this planet. Even though, the Simpsons made a joke about him once, too.

> Anyway, I'm feed up with this stuff. It was a small simple joke, and Cookie took it to an extreme and it went over his head. Now he's blaming mostly the wrong people; all of GH, twitter, and 4chan is wrong. This is nonsense, but it seems to be over now, I hope.

And yet, even though you want it "over", you came here and justified bad behaviour of bad people and blamed the victim, who has lost the most over this, for YOUR behaviour. ARE YOU SERIOUS? are you really THAT derailed from all the postings on 4chan? I am speechless. And that with so many words.

**Reply by differentprogramming** [reply]
Oh God, don't try this hard to shame 4chan.  Nothing good will come of this.

**Reply by differentprogramming** [reply]
Imagine that you're arguing with a troll.  Waste of time?

Now imagine that you're picking a fight with 100 anonymous kids.  Sorry for being amused and posting about it.

**Reply by ghost** [reply]
Holy shit dude, calm the fuck down. I wanted to handle this in an open minded way, not some delusional flame war that never ends.

> Wow. Another example of a sick mind. What else do you find "funny"? Something about Jews, by chance? You must be a troll. Or a asshole. Probably both. And a idiot.

I don't, it was an example lad. Why does "kittens with aids" bother you that much.

> Soo.... it is okay, because a well researched dictionary is wrong

I don't even know where to laugh. It's urban dictionary, lmao

> You make the assumption, that people are incapable of making more than one call ever. News flash: Technology exists. Computer can be programmed. Robocalls, speed-dialing and Automation are a thing for over 30 years

Nobody even discussed or new his phone number.

> And it was rejected. Deal with it like a grown-up.

Then why didn't he make the grown-up option and choose the second one?

Anyway, it's late. You cherry picked nearly every sentence I sprouted so I don't the time to read each take. You took it too seriously. Goodnight

**Comment by User-green** [comment]
IMO, this thread is made for reasons of trolling, and this way should be removed.

If anyone from 4chan realy whants to disscus things like "How engineer is horrible in person", than thay already have /b/. Github, IMO, should stay clear of accusations and trolling.

**Comment by Hyenadae** [comment]
Sneedacity is a fork of peace, the maintainers of Sneedacity are not represented by the general 4chan population. Please do not marginalize the Audacity forks, and use your bigotry against anonymous websites to remove or ruin Open Source software. 

#IStandWithSneed

**Comment by Semisol** [comment]
Issue locked due to inactivity + potentially controvertial.
If you want to request an unlock, please message me on [Libera.chat](//libera.chat) IRC or [DM me on Matrix](//matrix.to/#/@semisol:semisol.dev)
